pet_wellness_app1/app/javascript/components/pages/Landing.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
```

Comment:
Here you are importing Component with React but never use Component and instead use Fragment. 
```suggestion
import React, { Fragment } from 'react'
```